src/java.xml/share/classes/com/sun/org/apache/xerces/internal/impl/dv/DatatypeException.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2017, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Update the LastModified tag below as well. Otherwise, looks good.

Reply:
Done, thanks.